packages/legend-studio/src/stores/EditorStore.ts

```diff
@@ -790,24 +791,39 @@ export class EditorStore {
     this.initState.setMessage(`Fetching entities...`);
     try {
       // fetch workspace entities and config at the same time
+      const projectId = this.sdlcState.activeProject.projectId;
+      const activeWorkspace = this.sdlcState.activeWorkspace;
       const result = (yield Promise.all([
-        this.sdlcServerClient.getEntities(
-          this.sdlcState.activeProject.projectId,
-          this.sdlcState.activeWorkspace,
-        ),
-        this.sdlcServerClient.getConfiguration(
-          this.sdlcState.activeProject.projectId,
-          this.sdlcState.activeWorkspace,
-        ),
+        this.sdlcServerClient.getEntities(projectId, activeWorkspace),
+        this.sdlcServerClient.getConfiguration(projectId, activeWorkspace),
       ])) as [Entity[], PlainObject<ProjectConfiguration>];
       entities = result[0];
       projectConfiguration = result[1];
+
+      const projectConfigurationPayload = projectConfiguration;
+      const projectDependencyList =
+        projectConfiguration.projectDependencies as {
```

Comment:
Do we need this?
My understanding was going forward legend-sdlc will always give us thew new format when we ask for the dependency list. 
Hence it will give us versionId: string. 
It will be on us to understand this versionId string

Reply:
@akphi pretty sure my statement here is corret but could you verify

Reply:
yeah, I'm looking at @kevin-m-knight-gs PR https://github.com/finos/legend-sdlc/pull/442/files - https://github.com/finos/legend-sdlc/pull/442/files#diff-33676c119eb36f75a50e53a37b8e1359783afd139a84f0c985900a26c768e2c9

and I could see now `ProjectDependency` returns a string instead

Reply:
That means you could also remove `convertVersionObjectToString` method

Reply:
How did you test this out btw?

Reply:
I suspected that could be from an older SDLC server. With the new code, I don't really see how the version could be `non-string`

Reply:
@ijeh-i could you do more testing then? If you don't find any case, let's proceed assuming that it's string only

Reply:
ok ... I'll test again and confirm